apps/web/playwright/reschedule.e2e.ts

```diff
@@ -416,6 +418,112 @@ test.describe("Reschedule Tests", async () => {
+        attendees: {
+          createMany: {
+            data: nonCalUserAttendees,
+          },
+        },
+      });
+
+      await page.goto(`/reschedule/${booking.uid}`);
+
+      // Let current month dates fully render.
+      await page.click('[data-testid="incrementMonth"]');
+      // Waiting for full month increment
+      await page.locator('[data-testid="day"][data-disabled="false"]').nth(0).click();
+      // Expect normal full availability which 32 time slots
+      await expect(page.locator('[data-testid="time"]')).toHaveCount(32);
+    });
+    test("host reschedule with full cal.com user guests should trigger guests availability awareness", async ({
+      page,
+      users,
+      bookings,
+    }) => {
+      const host = await users.create();
+      const guests = await Promise.all(
+        Array.from({ length: 10 }, (_, i) => users.create({ username: `guest-${i}` }))
+      );
+      const eventType = host.eventTypes[0];
+      const booking = await bookings.create(host.id, host.username, eventType.id, {
+        attendees: {
+          createMany: {
+            data: guests.map((guest, idx) => ({
+              email: guest.email,
+              name: guest.username ?? `Guest ${idx}`,
+              timeZone: "Europe/London",
+            })),
+          },
+        },
+      });
+
+      await setGuestWorkingHours12PM5PM({ page, host, guest: guests[0] });
+
+      await page.goto(`/reschedule/${booking.uid}`);
+
+      // Let current month dates fully render.
+      await page.click('[data-testid="incrementMonth"]');
+      // Waiting for full month increment
+      await page.locator('[data-testid="day"][data-disabled="false"]').nth(0).click();
+      // Expect modified full availability which 8 time slots
+      await expect(page.locator('[data-testid="time"]')).toHaveCount(8);
+    });
+    test("host reschedule with mixed cal.com and non-cal.com user guests should trigger guests availability awareness", async ({
+      page,
+      users,
+      bookings,
+    }) => {
+      const host = await users.create();
+      const guests = await Promise.all(
+        Array.from({ length: 10 }, (_, i) => users.create({ username: `guest-${i}` }))
+      );
+      const nonCalUserAttendees = Array.from({ length: 5 }, (_, i) => ({
+        email: `attendee-${i}@example.com`,
+        name: `Attendee Example ${i}`,
+        timeZone: "Europe/London",
+      }));
+      const eventType = host.eventTypes[0];
+      const booking = await bookings.create(host.id, host.username, eventType.id, {
+        attendees: {
+          createMany: {
+            data: [
+              ...guests.map((guest, idx) => ({
+                email: guest.email,
+                name: guest.username ?? `Guest ${idx}`,
+                timeZone: "Europe/London",
+              })),
+              ...nonCalUserAttendees,
+            ],
+          },
+        },
+      });
+
+      await setGuestWorkingHours12PM5PM({ page, host, guest: guests[0] });
```

Comment:
Can it be done via user.create fixture instead? Not really a big fan of using e2e to do it, they are very flaky plus time consuming.

Reply:
nope, currently we can't do it using user fixture